features-json/av1.json

```
@@ -498,7 +498,7 @@
   },
   "notes":"",
   "notes_by_num":{
-    "1":"Can be enabled in Edge 18 by installing the AV1 Video Extension from the Microsoft Store",
+    "1":"Can be enabled in Edge under windows 10 and windows 11 by installing the AV1 Video Extension from the Microsoft Store",
```

Wording improvement suggestion:

```suggestion
    "1":"Can be enabled in Edge on Windows 10 or later by installing the [AV1 Video Extension from the Microsoft Store](https://www.microsoft.com/store/productId/9MVZQVXJBQ9V)",
```